.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,18 +13,17 @@ A clear and concise description of what the bug is.
 **Affected platforms**
 - [ ] All
 - [ ] Desktop
-- [ ] web (k/js) - html based API
-- [ ] web (k/js) - Canvas based API
-- [ ] k/native
+- [ ] Web (K/JS) - HTML based API
+- [ ] Web (K/JS) - Canvas based API
+- [ ] iOS
 - [ ] other
 
 **Versions**
 - Kotlin version*: 
-- Compose multiplatform version*:
-- OS version(s)* (required for desktop and k/native issues):
+- Compose Multiplatform version*:
+- OS version(s)* (required for Desktop and iOS issues):
 - OS Architecture (x86 or ARM64): 
 
-
 **To Reproduce**
 Steps and/or the code snippet to reproduce the behavior:
 1. Go to '...'
```

Comment:
Should we add an article in `4. See error`? (I don't know :) )

Reply:
I always have doubts about articles)
In this case I took those steps from the default github template, so... I still don't know)

Reply:
If it is a default, it should be good